docker/ccd-definition-import/scripts/upload-definition.sh

```diff
@@ -3,25 +3,18 @@ set -e
 
 chmod +x /scripts/*.sh
 
-IMPORTER_USERNAME=ccd-importer@server.net
-IMPORTER_PASSWORD=Password12
-IDAM_URI=http://idam-api:8080
-REDIRECT_URI=http://localhost:3000/receiver
-CLIENT_SECRET=123456
-BULK_SCAN_ORCHESTRATOR_BASE_URL=http://host.docker.internal:8582
-
 if [ ${VERBOSE} = "true" ]; then
   export CURL_OPTS="-v"
 else
   export CURL_OPTS="--fail --silent"
 fi
 
-/scripts/create-importer-user.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD}
+[ "_${CREATE_IMPORTER_USER}" = "_yes" ] && /scripts/create-importer-user.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD}
```

Comment:
why the `_`

Reply:
Precaution in case the variable is not set at all (not to get an ugly error message).